eclair-core/src/main/scala/fr/acinq/eclair/channel/Commitments.scala

```diff
@@ -458,7 +458,7 @@ case class Commitment(fundingTxIndex: Long,
     } else if (missingForReceiver < 0.msat) {
       if (params.localParams.isInitiator) {
         // receiver is not the channel initiator; it is ok if it can't maintain its channel_reserve for now, as long as its balance is increasing, which is the case if it is receiving a payment
-      } else if (reduced.toLocal > fees && reduced.htlcs.size < 5) {
+      } else if (reduced.toLocal > fees && reduced.htlcs.size < 5 && fundingTxIndex > 0) {
```

Comment:
Shouldn't we test channel features instead?

Reply:
What exactly would you test? We can't rely on just dual-funding, because that will soon include channels with `cln` nodes that will have the same compatibility issue for the reserve.

Reply:
Nevermind!